Redirect home page (Early stage)

filipkrueger opened the pull request.

filipkrueger committed:
Redirect home page (Early stage)

gustavwogen commented on README.md:
```diff
@@ -12,4 +12,6 @@
 	- Create necessary database tables for our application
 	- Populate those tables with dummy data
 
+- Run `npm init` to install all packages required
```
Should be run npm install

gustavwogen committed:
Update README.md

should be npm install when there already is a package.json you just want to install everything from
